userguide/tasks.rst

```diff
@@ -1745,14 +1750,14 @@ shows the configuration options for Cloud Syncs.
    | Setting             | Value Type          | Description                                                                                             |
    |                     |                     |                                                                                                         |
    +=====================+=====================+=========================================================================================================+
-   | Description         | string              | Enter a descriptive name of this task.                                                                  |
+   | Description         | string              | Enter a description of the Cloud Sync Task.                                                             |
    |                     |                     |                                                                                                         |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
    | Direction           | drop-down menu      | *Push* sends data to cloud storage. *Pull* receives data from cloud storage.                            |
    |                     |                     |                                                                                                         |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
-   | Credential          | drop-down menu      | Choose the cloud storage provider credentials from the list of entered :ref:`Cloud Credentials`. The    |
-   |                     |                     | UI automatically tests the credential and displays an error if a connection cannot be made. The         |
+   | Credential          | drop-down menu      | Select the cloud storage provider credentials from the list of available :ref:`Cloud Credentials`.      |
+   |                     |                     | The UI automatically tests the credential and displays an error if a connection cannot be made.         |
```

Comment:
"The credential is tested and an error is displayed if a connection cannot be made."

Reply:
done